config/application.rb

```diff
@@ -28,3 +28,4 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
   end
 end
+
```

Comment:
One last place: Same thing as application.js here -- lets revert this file to be unchanged.